doc/source/whatsnew/v1.4.0.rst

```diff
@@ -543,6 +543,7 @@ Other Deprecations
 - Deprecated :meth:`DateOffset.apply`, use ``offset + other`` instead (:issue:`44522`)
 - Deprecated parameter ``names`` in :meth:`Index.copy` (:issue:`44916`)
 - A deprecation warning is now shown for :meth:`DataFrame.to_latex` indicating the arguments signature may change and emulate more the arguments to :meth:`.Styler.to_latex` in future versions (:issue:`44411`)
+- Deprecated passing arguments as positional in :meth:`DataFrame.any` and :meth:`Series.any` (:issue:`44802`)
```

Comment:
Which issue number should I mention. 
The issue:44802 doesn't give context for a reason to deprecate.

Reply:
I think people often simply use the PR number if there is no specific issue. I am not sure whether this is good practice.

Reply:
IMO the issue's fine, people can find the PR from there